py_hamt/hamt_to_sharded_converter.py

```
@@ -0,0 +1,126 @@
+import argparse
+import asyncio
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)